core/domain/collection_domain.py

```diff
@@ -49,8 +49,8 @@
 # This takes additional 'property_name' and 'new_value' parameters and,
 # optionally, 'old_value'.
 CMD_EDIT_COLLECTION_NODE_PROPERTY = 'edit_collection_node_property'
-# This takes additional 'from_version' and 'to_version' parameters for logging.
-CMD_MIGRATE_SCHEMA_TO_LATEST_VERSION = 'migrate_schema_to_latest_version'
+# This takes additional 'version' parameter for logging.
+CMD_MIGRATE_SCHEMA = 'migrate_schema'
```

Comment:
See my comment above -- the collection migration structure is somewhat different from explorations, in that it performs any necessary migrations on reading the collection from the model, and not on save like in explorations. So, it doesn't make sense to have a 'from_version' and 'to_version'. Furthermore, I believe there have currently not been any migrations done (since this migration job is new), so changing it is ok.